browser/resources/settings/brave_overrides/settings_menu.ts

```diff
@@ -190,12 +189,12 @@ RegisterStyleOverride(
         flex-direction: row;
         align-items: flex-start;
         justify-content: flex-start;
-        color: #c5c5d3 !important;
+        color: var(--leo-color-text-tertiary) !important;
         margin: 16px 0 0 0 !important;
       }
       .brave-about-graphic {
         flex: 0;
-        flex-basis: 30%;
+        flex-basis: var(--leo-spacing-3xl);
```

Comment:
what does this do?

Reply:
Before, it was setting the padding between the logo and the text in a % of the whole menu width, super weird behavior. I set it so it's fixed and keeps the same alignment as the nav buttons